epregressions/tests/resources/README.md

```diff
@@ -2,16 +2,15 @@ This folder contains a bunch of resources for running tests.
 These are extremely lightweight executable scripts that mimic the actual E+ toolchain.
 There are also other files in here such as example weather files and E+ run files.
 
-def execute_energyplus(build_tree, entry_name, test_run_directory,
-                       run_type, min_reporting_freq, this_parametric_file, weather_file_name):
-    # setup a few paths
-    energyplus = build_tree['energyplus']
-    basement = build_tree['basement']
-    idd_path = build_tree['idd_path']
-    slab = build_tree['slab']
-    basementidd = build_tree['basementidd']
-    slabidd = build_tree['slabidd']
-    expandobjects = build_tree['expandobjects']
-    epmacro = build_tree['epmacro']
-    readvars = build_tree['readvars']
-    parametric = build_tree['parametric']
+The windows versions of these are prebuilt using pyinstaller and placed in the `dist/` directory
```

Comment:
This was a huge pain, pyinstaller is just flaky as crap, I could rerun the command multiple times and it would fail about half the time.